src/content/en/updates/2019/10/devtools.md

```diff
@@ -0,0 +1,81 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2019-10-14 #}
+{# wf_published_on: 2019-10-14 #}
+{# wf_tags: chrome79, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 79) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Cookie updates {: #cookies }
+
+### Blocked cookies {: #blockedcookies }
+
+[Request headers](https://chromium.googlesource.com/chromium/src/+/f82c0c2418170ac32a9764550073bc3c95dce240),
+[Cookies tab in Network panel](https://chromium.googlesource.com/chromium/src/+/4f13a2518b054a7e8d218aaf3213de2e1c74ea4e),
+[Multiple explanations of blocked cookies](https://chromium.googlesource.com/chromium/src/+/8a8d484654bf7cba22df22a9dc072e49bcb73327)
+
+[Chromium issue #856777](https://crbug.com/856777)
+
+### Cookie value previews {: #cookiepreviews }
+
+[Show values when selected](https://chromium.googlesource.com/chromium/src/+/2c327126ab78b37dfbcedee2a8a267d2a54dadfc)
+
+## Initiator tab {: #initiator }
+
+[Network request initiator stack trace](https://chromium.googlesource.com/chromium/src/+/0d0cae574b458c3d4bd7f6f254f74f6f2442b710)
+
+## Emulate CSS media features
+
+[prefers-color-scheme: dark](https://chromium.googlesource.com/chromium/src/+/9ca30329b8ee53b2462d72772dc189385b6e2a34)
+
+## Code coverage
```

Comment:
I think there were some other minor tweaks being done by @sigurdschneider e.g. clicking on the empty coverage gutter will display the Coverage panel etc.

Reply:
I'll check the commit log again to see if there's other stuff to cover

Reply:
I mentioned the gutter feature